toolchain/parse/node_kind.def

```diff
@@ -70,6 +70,30 @@ CARBON_PARSE_NODE_KIND_CHILD_COUNT(Name, 0,
 // A name in an expression context.
 CARBON_PARSE_NODE_KIND_CHILD_COUNT(NameExpression, 0, CARBON_TOKEN(Identifier))
 
+// ----------------------------------------------------------------------------
+
+// The comments below follow this pattern:
+
+// Descriptive heading:
+//   Child1
+//   Child2
+// Parent
+
```

Comment:
```suggestion
//
// // Descriptive heading:
// //   Child1
// //   Child2
// // Parent
//
```

I think using a contiguous block comment makes context clearer; the colon at the end of line 77 is a little confusing otherwise, matching the colon at the end of line 75.